# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gfx-backend-vulkan/README.md
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gfx-backend-vulkan/README.md b/third_party/rust/gfx-backend-vulkan/README.md
deleted file mode 100644
--- a/third_party/rust/gfx-backend-vulkan/README.md
+++ /dev/null
@@ -1,20 +0,0 @@
-# gfx-backend-vulkan
-
-[Vulkan](https://www.khronos.org/vulkan/) backend for gfx-rs.
-
-## Normalized Coordinates
-
-Render | Depth | Texture
--------|-------|--------
-![render_coordinates](../../../info/vk_render_coordinates.png) | ![depth_coordinates](../../../info/dx_depth_coordinates.png) | ![texture_coordinates](../../../info/dx_texture_coordinates.png)
-
-## Binding Model
-
-Dimensions of the model:
-  1. Shader stage: vs, fs, cs, others
-  2. Descriptor set: 0 .. `max_bound_descriptor_sets`
-  3. Binding: sparse, but expected to be somewhat tight
-
-## Mirroring
-
-HAL is modelled after Vulkan, so everything should be 1:1.